truss/templates/trtllm/packages/schema.py

```diff
@@ -22,6 +22,8 @@ def __init__(
         ignore_eos: bool = False,
         stream: bool = True,
         eos_token_id: int = None,  # type: ignore
+        # enables new arguments to be passed in without breaking instantiation
```

Comment:
maybe leave a comment here that these arguments won't actually be passed to the triton server (because we don't actually convert these values below), they're just there to prevent breakage.